From 8b5da21c41ace2f9801de44841be7a125097e684 Mon Sep 17 00:00:00 2001
From: Owen Smith <yeeetari@gmail.com>
Date: Thu, 12 Aug 2021 22:29:41 +0100
Subject: [PATCH 4/4] patch time

---
 libiberty/gettimeofday.c | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/libiberty/gettimeofday.c b/libiberty/gettimeofday.c
index fca16794..fb690a7d 100644
--- a/libiberty/gettimeofday.c
+++ b/libiberty/gettimeofday.c
@@ -1,11 +1,7 @@
 #include "config.h"
 #include "libiberty.h"
-#ifdef HAVE_TIME_H
 #include <time.h>
-#endif
-#ifdef HAVE_SYS_TIME_H
 #include <sys/time.h>
-#endif
 
 /* 
 
-- 
2.32.0

